internal/errors/blob_test.go

```diff
@@ -120,31 +104,33 @@ func TestIsErrBlobNoSuchBucket(t *testing.T) {
 		return nil
 	}
 	tests := []test{
-		// TODO test cases
-		/*
-		   {
-		       name: "test_case_1",
-		       args: args {
-		           err: nil,
-		       },
-		       want: want{},
-		       checkFunc: defaultCheckFunc,
-		   },
-		*/
-
-		// TODO test cases
-		/*
-		   func() test {
-		       return test {
-		           name: "test_case_2",
-		           args: args {
-		           err: nil,
-		           },
-		           want: want{},
-		           checkFunc: defaultCheckFunc,
-		       }
-		   }(),
-		*/
+		{
+			name: "returns true when the error is ErrBlobNoSuchBucket",
+			args: args{
+				err: &ErrBlobNoSuchBucket{New("nosuchbucketerr")},
+			},
+			want: want{
+				want: true,
+			},
+		},
+		{
+			name: "returns false when the error is not ErrBlobNoSuchBucket",
+			args: args{
+				err: New("anothererr"),
```

Comment:
```suggestion
				err: &ErrBlobNoSuchKey{New("no such key err")}
```
how about it?
i think it'd be more meaningful 🤔